README.md

```diff
@@ -53,6 +53,8 @@ change as part of the work.
 `basic_search#index`.
 - `GLOBAL_ALERT`: The main functionality for this comes from our theme gem, but when set the value will be rendered as
   safe html above the main header of the site.
+- `SENTRY_DSN`: Client key for Sentry exception logging.
+- `SENTRY_ENV`: Sentry environment for the application. Defaults to 'prod' if unset.
```

Comment:
I wonder if we should have the default be something like `unknown` to clarify that we don't actually _know_ if it's in prod unless it's configured. I can see either side, but feel like it's worth considering.

Reply:
Yeah, I thought about `unknown` as well. If you're good with that, I'll go ahead and make that change.

Reply:
Looks like we updated the method but not the docs on that last change